GetandTake/Core/Helper/FileHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace GetandTake.Core.Helper;
+
+public class FileHelper
+{
+    public static string Add(IFormFile file)
+    {
+        var result = NewPath(file);
+        try
+        {
+            var sourcePath = Path.GetTempFileName();
+            if (file.Length > 0)
+            {
+                using (var stream = new FileStream(sourcePath, FileMode.Create))
+                {
+                    file.CopyTo(stream);
+                    File.Move(sourcePath, result.NewFilePath);
+                }
+            }
+        }
+        catch (Exception exception)
+        {
+
```

Comment:
Well, if you use `curly` brackets and there is only one `return` statement, then no need to add empty row:
```suggestion
catch (Exception exception)
{
    return exception.Message;
}
```

But, if you have some code with `return` statement, than empty row should be:
```suggestion
catch (Exception exception)
{
    var a = 1;
    var b = 2;
    ...

    return exception.Message;
}
```